ci/tasks/test-api.sh

```diff
@@ -3,6 +3,7 @@ set -euo pipefail
 
 cd app
 
-./scripts/wait-for-it.sh db:5432 -- yarn test:cover; status=$?
-
-exit $status
+# ./scripts/wait-for-it.sh db:5432 -- yarn test:cover; status=$?
+echo "we're rewriting our tests!"
+exit 0
```

Comment:
Let's update this so we still run the server tests.

```suggestion
./scripts/wait-for-it.sh db:5432 -- yarn test:prepare && yarn test:server:cover
exit $status
```